### ISSUE TITLE
[processor/memorylimiter] clarification abour memory limiter behavior in mixed pipelines

### ISSUE THREAD

**Issue by hilshalom** [open-telemetry/opentelemetry-collector#9006]
Hi, I am conducting a research, and would like to configure memory limiter only on some of the pipelines.
For my understanding pipelines without memory limiter will not reject any resources when the memory limit is reached by a another pipeline, and won't experience any resource loss during memory limited mode. Is that correct?
I would appreciate your response and clarification.
Thank you

**Comment by dmitryax** [comment]
> For my understanding pipelines without memory limiter will not reject any resources when the memory limit is reached by a another pipeline, and won't experience any resource loss during memory limited mode. Is that correct?

That's correct. Keep in mind that the collector can run out of memory and crash in that case
